### PR TITLE
Fix loan

### PR DESCRIPTION
**Pull request by TwistedHead**
This is ready for review.

### CODE REVIEW

File: src/Loan.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	private book book;
+	private member member;
+	private Date date;
+	private LoanState state;
+
+	public Loan(int loanId, book book, member member, Date dueDate) {
+		this.id = loanId;
+		this.book = book;
+		this.member = member;
+		this.date = dueDate;
```
Comment (jazzie-b) [inline src/Loan.java]: Tabs used instead of spaces for indentation.

File: src/Loan.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		this.state = LoanState.CURRENT;
+	}
+
```
Comment (jazzie-b) [inline src/Loan.java]: Method should have 2 line spacing. Issue throughout

Comment (TwistedHead) [reply src/Loan.java]: updated method names with 2 line spacing.

File: src/Loan.java (hunk 1 above)
Comment (TwistedHead) [reply src/Loan.java]: Updated as requested.

**Comment by TwistedHead** [comment]
has too many conflicts. Closing and creating a new branch to fix loan.

**Pull request closed without merging**